### PR TITLE
Release/1.1.0

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
1. Added Configuration Step in both PB to notify for IOCs Count.
2. Added Exchange Connector to SP.
3. Linked new and existing Indicator extracted from file to parent Indicator(Over which PB is executed).
4. Alert was getting linked with new indicator only not with the existing, so removed linking of new indicator to alert.

### CODE REVIEW

File: playbooks/globalVariables.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
 [
+    {
+        "id": 7,
+        "name": "Server_fqhn",
+        "value": "master.fortisoar.in",
```
Comment (cs-suyog-jinturkar) [inline playbooks/globalVariables.json]: We don't need these global variables as part of this SP

Comment (cs-suyog-jinturkar) [reply playbooks/globalVariables.json]: Please check with Rohit once

File: info.json [hunk 2]
```diff
@@ -9,32 +9,44 @@
...
         "playbooks": [
             {
-                "name": "02 - Use Case - File Content Extraction - 1.1.0"
+                "name": "02 - Use Case - File Content Extraction"
             }
         ],
         "globalVariables": [
+            {
+                "id": 7,
+                "name": "Server_fqhn",
```
Comment (cs-suyog-jinturkar) [inline info.json]: Global variables are not needed here

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comments

File: playbooks/globalVariables.json (hunk 1 above)
Comment (cs-rohit-khune) [reply playbooks/globalVariables.json]: @cs-ankit-gupta @cs-suyog-jinturkar "Server_fqhn" global variable can be ignored

File: README.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+# Overview
+
+Solution Pack examines Advisories sent by various organisations via email with attachments in PDF, Excel, CSV, and other formats. `File Metadata` and `File Content` is added to the record's Description. It notifies the count of IPs, Domain Name, Hashes, URLs, and so on if they are discovered via Email
+
+![](./docs/res/Description.png)
+
+
+# Next Steps
+
+| [Installation](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/setup.md#installation) | [Configuration](https://github.com/solution-pack-file-content-extraction/blob/release/1.1.0/docs/setup.md#configuration) | [Usage](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/usage.md) | [Contents](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/contents.md) |
```
Comment (cs-rohit-khune) [inline README.md]: Doc URL of [Configuration] is wrong, `fortinet-fortisoar` is missing. @cs-ankit-gupta Please correct it

Comment (cs-rohit-khune) [reply README.md]: Also, add release_notes.md
@cs-ankit-gupta

File: docs/contents.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+# Contents
+
+The **File Content Extraction** solution pack contains the following resources.
+
+## Connectors
+
+|**Name**|**Description**|
+| :- | :- |
+| Exchange | The Exchange connector provides a robust, platform-independent, and simple interface for communicating with Microsoft Exchange 2007-2016 Server or Office 365 using Exchange Web Services (EWS). |
+| File Content Extraction | The connector uses is used to extract content, metadata and artifacts from over 1500 file types such as MS Office, PDF...etc |
```
Comment (cs-rohit-khune) [inline docs/contents.md]: Please improve the File Content Extraction description sentence 

`The connector uses is used to extract content`

File: info.json [hunk 5]
```diff
@@ -52,13 +64,23 @@
                 "name": "Excludelist_Domains",
                 "value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com,fortiguard.com,forticloud.com,fortielab.com",
                 "default_value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com,fortiguard.com,forticloud.com"
+            },
+            {
+                "id": 33,
+                "name": "Excludelist_Emails",
+                "value": "ena.vuk@bhtelecom.ba,sysadmin@bhtelecom.ba",
```
Comment (cs-rohit-khune) [inline info.json]: @cs-ankit-gupta Any specific reason, why these emails are used in Excludelist_Emails?

File: playbooks/02 - Use Case - File Content Extraction - 1.1.0/Extract and Process Text From Attachment File.json [hunk 6]
```diff
@@ -1,288 +0,0 @@
-{
```
Comment (cs-rohit-khune) [inline playbooks/02 - Use Case - File Content Extraction - 1.1.0/Extract and Process Text From Attachment File.json:1]: Remove playbooks/.DS_Store

Comment (cs-rohit-khune) [reply playbooks/02 - Use Case - File Content Extraction - 1.1.0/Extract and Process Text From Attachment File.json:1]: @cs-ankit-gupta

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]

File: playbooks/02 - Use Case - File Content Extraction - 1.1.0/Extract and Process Text From Attachment File.json (hunk 6 above)
Comment (cs-ankit-gupta) [reply playbooks/02 - Use Case - File Content Extraction - 1.1.0/Extract and Process Text From Attachment File.json:1]: Done

File: info.json (hunk 5 above)
Comment (cs-ankit-gupta) [reply info.json]: These were the default values

File: playbooks/globalVariables.json (hunk 1 above)
Comment (cs-suyog-jinturkar) [reply playbooks/globalVariables.json]: Ignored means, can be removed. What about other global variables? @cs-rohit-khune

File: README.md (hunk 3 above)
Comment (cs-ankit-gupta) [reply README.md]: Added release_notes.md

File: docs/contents.md (hunk 4 above)
Comment (cs-ankit-gupta) [reply docs/contents.md]: modified

File: info.json (hunk 2 above)
Comment (cs-ankit-gupta) [reply info.json]: Removed all Global Variables from info.json

File: playbooks/globalVariables.json (hunk 1 above)
Comment (cs-ankit-gupta) [reply playbooks/globalVariables.json]: Remved all global variable from globalVariables.json

File: info.json [hunk 7]
```diff
@@ -9,56 +9,41 @@
...
-            "version": "1.0.0",
+            "version": "1.1.0",
             "minVersion": null
         }
     ],
     "prerequisite": null,
-    "publisher": "Community",
-    "description": "Playbooks to extract file metadata and content. The result is added to the record's description",
-    "help": null,
+    "publisher": "Fortinet SE",
```
Comment (cs-suyog-jinturkar) [inline info.json:17]: CSE

File: info.json [hunk 8]
```diff
@@ -9,56 +9,41 @@
...
+    "description": "Playbooks to extract file metadata and content. The result is added to the record's description and notify the count of IOCs",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/README.md",
     "category": [
-        "Investigation",
-        "Enrichment"
+        "Enrichment",
```
Comment (cs-suyog-jinturkar) [inline info.json]: enrichment is a wrong category. It Extracts the IOC. @cs-rohit-khune  what should be the correct name here?

Comment (cs-rohit-khune) [reply info.json]: IMO `Utilities` would be better @cs-suyog-jinturkar

File: info.json (hunk 7 above)
Comment (cs-ankit-gupta) [reply info.json:17]: Done

**Review by cs-rohit-khune — APPROVED** [review]
LGTM

**Pull request merged**